tests/fake/test_fake_search_content.py

```diff
@@ -32,6 +32,9 @@ def populated_repo(controller):
             sourcerpm="glibc-5.0-1.el5_11.1.src.rpm",
             provides=[RpmDependency(name="gcc")],
         ),
+        RpmUnit(
+            name="jq", version="4.0", release="1", arch="x86_64", files=["/some/file"]
```

Comment:
Personally I think it would be a good idea to make the files list contain more than one entry.  There was already some confusion while working on this PR about how searching for a value within a list field works.  Having the list contain only a single value raises the possibility that maybe the code or the test is *only* working in the case where the list has a single value, which is not realistic.

In other words, maybe there is a bug where `Criteria.with_field("files", "/some/file")` only matches an RpmUnit having files equal to *exactly* `["/some/file"]`, but the tests wouldn't be able to expose that bug right now.

Reply:
Yeah, good idea